crates/volta-core/src/tool/mod.rs

```diff
@@ -193,6 +194,24 @@ impl ToolSpec {
     }
 }
 
+/// Function for comparing toolspecs for use in sorting.
```

Comment:
Formatting nits:

-   We can make the first line imperative voice:

    ```rust
    /// Compare `ToolSpec`s for sorting.
    ```
  
    This drops the repetitive "Function for…" at the beginning


-   There should be a blank newline (starting a new Markdown paragraph) between the first line (the summary) and the following lines.

-   My own personal preference, which helps a lot with readability in apps which don't have some kind of autowrapping (or have it disabled, as I do), is to wrap paragraphs at 80 characters, so:

    ```rust
    /// We want to preserve the original order as much as possible, so we treat same
    /// tool categories as equal. We still need to pull Node up first, followed by
    /// Npm / Yarn, and then Packages last.
    ```

    (That one is *really* just a nit, though.)

Reply:
Done!